src/test/java/org/jabref/logic/integrity/EditionCheckerTest.java

```diff
@@ -0,0 +1,23 @@
+package org.jabref.logic.integrity;
+
+import org.jabref.model.database.BibDatabaseContext;
+import org.junit.jupiter.api.Test;
+
+
```

Comment:
Please follow our code style -- no multiple empty lines.

Reply:
Fixed it